browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -172,34 +175,56 @@ BraveBrowserView::BraveBrowserView(std::unique_ptr<Browser> browser)
                           base::Unretained(this)));
 #endif
 
+  const bool can_have_vertical_tabs =
+      browser_->is_type_normal() &&
+      base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs);
+  const bool can_have_sidebar = sidebar::CanUseSidebar(browser_.get());
```

Comment:
`CanUseSidebar` needs to be guarded by `ENABLE_SIDEBAR`. This should be made more clear by adding `assert(enable_sidebar)` to `browser/ui/sidebar/BUILD.gn` but the include for sidebar_utils.h in this file is behind the flag so this needs to be as well.